pkgs/kubernetes/kubernetes/kube-aggregator/scaffold.yaml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/aqua-generate-registry.json
+# aqua - Declarative CLI Version Manager
+# https://aquaproj.github.io/
+# Other than name is optional. All initial values are just examples.
+name: kubernetes/kubernetes/kube-aggregator
+version_filter: not (Version matches "-(alpha|beta|rc)")
+# version_prefix: cli-
+# all_assets_filter: not (Asset matches "-cli")
```

Comment:
scaffold.yaml is unnecessary.
`cmdx s` supports only `github_release` type package.

Reply:
:bow: